base_layer/tari_mining_helper_ffi/src/lib.rs

```diff
@@ -233,6 +245,153 @@ pub unsafe extern "C" fn inject_nonce(header: *mut ByteVector, nonce: c_ulonglon
     (*header).0 = buffer;
 }
 
+/// Injects a coinbase into a blocktemplate
+///
+/// ## Arguments
+/// `block_template_bytes` - The block template as bytes, serialized with borsh.io
+/// `value` - The value of the coinbase
```

Comment:
If this includes the fee it should be like `reward_and_fee` or something and the comment should explain it